xc/device.mli

```diff
@@ -21,6 +21,20 @@ exception Device_not_found
 
 exception Cdrom
 
+module Profile: sig
+	type t = Qemu_trad | Qemu_upstream_compat | Qemu_upstream
+	val fallback : t
+	module Name: sig
+		val qemu_trad: string
+		val qemu_upstream_compat: string
+		val qemu_upstream: string
+		val all: string list
```

Comment:
It's better to have `val all: t list` and then use `List.map to_string all`, I believe.

Reply:
done